services/QuillLMS/app/models/app_setting.rb

```diff
@@ -0,0 +1,65 @@
+# == Schema Information
+#
+# Table name: app_settings
+#
+#  id                 :integer          not null, primary key
+#  enabled            :boolean          default(FALSE), not null
+#  enabled_for_admins :boolean          default(FALSE), not null
+#  name               :string           not null
+#  pct_active         :integer          default(0), not null
+#  user_whitelist     :string           default([]), not null, is an Array
+#  created_at         :datetime         not null
+#  updated_at         :datetime         not null
+#
+# Indexes
+#
+#  index_app_settings_on_name  (name)
+#
+class AppSetting < ActiveRecord::Base
+    PCT_ACTIVE_VALUES = [100, 90, 80, 70, 60, 50, 40, 30, 20, 10, 0] 
```

Comment:
I've noticed some of the ruby code has four spaces indentation instead of the standard two.  Is this intentional?

I think consistency helps with readability as four spaces usually think Python when I see that spacing.